lib/controllers/widget/hardware_os_card.dart

```diff
@@ -47,359 +48,92 @@ class _HardwareAndOsCardState extends State<HardwareAndOsCard>
-                          child: Column(
-                            crossAxisAlignment: CrossAxisAlignment.start,
-                            children: <Widget>[
-                              Text(
-                                'Auto Update Expiration',
-                                style: TextStyle(
-                                    fontSize: 18,
-                                    color: Colors.black87,
-                                    fontWeight: FontWeight.w500),
-                                overflow: TextOverflow.clip,
-                              ),
-                              Text(
-                                '${DateFormat.yMMMd().format(DateTime.fromMillisecondsSinceEpoch(int.parse(widget._device.autoUpdateExpiration)))}',
-                                style: TextStyle(
-                                    fontSize: 16, color: Colors.black54),
-                                overflow: TextOverflow.clip,
-                              )
-                            ],
-                          )),
+                      DetailedText(
+                          title: 'Organization',
+                          value: widget._device.orgUnitPath ?? '-'),
+                      DetailedText(
+                          title: 'Model', value: widget._device.model ?? '-'),
+                      DetailedText(
+                          title: 'Serial Number',
+                          value: widget._device.serialNumber ?? '-'),
+                      DetailedText(
+                          title: 'Mac Address',
+                          value: widget._device.macAddress ?? '-'),
+                      DetailedText(
+                        title: 'Last Known Network',
+                        value: widget._device.lastKnownNetwork == null
+                            ? '-'
+                            : widget._device.lastKnownNetwork
+                                .map((o) => o.ipAddress)
+                                .join('\n'),
+                      ),
+                      DetailedText(
+                          title: 'OS version',
+                          value: widget._device.osVersion ?? '-'),
+                      DetailedText(
+                          title: 'Plataform Version',
+                          value: widget._device.platformVersion ?? '-'),
+                      DetailedText(
+                          title: 'Device ID',
+                          value: widget._device.deviceId ?? '-'),
+                      DetailedText(
+                          title: 'Last Enrollment Time',
+                          value: widget._device.lastEnrollmentTime ?? '-'),
+                      DetailedText(
+                          title: 'Auto Update Expiration',
+                          value:
+                              '${DateFormat.yMMMd().format(DateTime.fromMillisecondsSinceEpoch(int.parse(widget._device.autoUpdateExpiration)))}'),
```

Comment:
What happens if this is not present? Shouldn't we also have the conditional '-' for this?